regexes/device/mobiles.yml

```diff
@@ -5832,6 +5891,17 @@ Leotec:
     - regex: 'LESPH5014'
       model: 'Krypton K150'
 
+# Lephone
+Lephone:
+  regex: 'lephone[ _]'
+  device: 'smartphone'
+  models:
+    - regex: 'lephone[ _](K7|K10)'
+      device: 'feature phone'
+      model: '$1'
+    - regex: 'lephone[ _](W[79])[/;) ]'
+      model: '$1'
```

Comment:
Maybe you could also add this one:
`Dalvik/v3.3.127 Compatible (TVM xx; YunOS 3.0; Linux; U; Android 4.4.4 Compatible; lephone T2 Build/KTU84P)`

Reply:
Added